src/exe/cimbar_send/send.cpp

```diff
@@ -87,7 +90,7 @@ int main(int argc, char** argv)
 		return true;
 	};
 
-	Encoder en(ecc);
+	Encoder en(ecc, cimbar::Config::symbol_bits(), colorBits);
```

Comment:
Another option is to make cimbar::Config ... run-time configurable. I might do that.